### PR TITLE
feat: Add after_order_message field in event

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #https://github.com/fossasia/open-event-frontend/issues/5560

required for #https://github.com/fossasia/open-event-frontend/pull/6388

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Screenshot of email and pdf please

**Comment by sachinchauhan2889** [comment]
email 

email


invoice 

invoice



ticket 

ticket

**Comment by sachinchauhan2889** [comment]
email

email


invoice

invoice


ticket

ticket

File: app/templates/email/ticket_purchased.html [hunk 1]
```diff
@@ -15,5 +15,9 @@
 <br/>{{ order.event.normalized_location }}
 <br/><br/>{{ _('Best regards') }},
 <br/>{{ settings.app_name }} {{ _('Team') }}
+{% if order.event.after_order_message %}
+    <br/><br/>Below you find a message from the organizer.
```
Comment (iamareebjamal) [inline app/templates/email/ticket_purchased.html:19]:
```suggestion
    <br/><br/>Below you find a message from the organizer:
```

File: app/templates/email/ticket_purchased.html [hunk 2]
```diff
@@ -15,5 +15,9 @@
 <br/>{{ order.event.normalized_location }}
 <br/><br/>{{ _('Best regards') }},
 <br/>{{ settings.app_name }} {{ _('Team') }}
+{% if order.event.after_order_message %}
+    <br/><br/>Below you find a message from the organizer:
```
Comment (iamareebjamal) [inline app/templates/email/ticket_purchased.html:19]: Make it translatable

File: app/templates/email/ticket_purchased_attendee.html [hunk 3]
```diff
@@ -13,5 +13,9 @@
 <br/>{{ order.event.normalized_location }}
 <br/><br/>{{ _('Best regards') }},
 <br/>{{ settings.app_name }} {{ _('Team') }}
+{% if order.event.after_order_message %}
+    <br/><br/>Below you find a message from the organizer.
```
Comment (iamareebjamal) [inline app/templates/email/ticket_purchased_attendee.html:17]: Same, and change . to :

File: app/templates/pdf/order_invoice.html [hunk 4]
```diff
@@ -245,6 +245,13 @@ <h1>INVOICE</h1>
         </tbody>
       </table>
       <br/>
+      {% if event.after_order_message %}
+        <div id="notices">
+          <div>Organizer Message:</div>
```
Comment (iamareebjamal) [inline app/templates/pdf/order_invoice.html:250]: translatable

**Pull request merged**